src/components/SingleAssetEntryExit.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+    import images from '../config/images.json'
+    import smartPools from '../config/pools.json'
+    import Modal from '../components/elements/Modal.svelte';
+    import Swap from '../components/JoinSwapExternAmount.svelte';
+    import { pools } from '../stores/eth.js'
+
+    export let buyTokenAddress;
+    export let openTrigger = () => swapModal.open();
+    let swapModal;
+
+    $: listed = $pools[buyTokenAddress.toLowerCase()];
+    $: symbol = smartPools[buyTokenAddress.toLowerCase()].symbol;    
+    
+</script>
+<section>
+    <slot>
+        <img src={images.defi_pp_merge} alt="defi single asset exchange"
+            class="cursor-pointer"
+            on:click={openTrigger}
+        />
+    </slot>
+    <Modal title={'Swap Tokens'} backgroundColor="#f3f3f3" bind:this={swapModal}>
+        <span class="w-full" slot="content">
```

Comment:
Do not wrap an entire component with divs inside spans (spans should only allow inline elements not blocks)

Reply:
In the svelte docs they use spans: https://svelte.dev/tutorial/named-slots